analyzers/src/SonarAnalyzer.CSharp/Rules/RedundantConditionalAroundAssignment.cs

```diff
@@ -19,13 +19,15 @@
  */
```

Comment:
I see that there is a codefix  provider for the rule. Lets try to update that too to work with c#9 patterns if it is a low hanging fruit.